frontend/public/module/k8s/k8s-models.ts

```diff
@@ -26,8 +30,7 @@ export const modelsToMap = (models: K8sKind[]): ImmutableMap<K8sResourceKindRefe
  */
 let k8sModels = modelsToMap(_.values(staticModels));
 
-const hasModel = (model: K8sKind) =>
-  k8sModels.has(referenceForModel(model)) || k8sModels.has(model.kind);
+const hasModel = (model: K8sKind) => k8sModels.has(getModelKey(model));
```

Comment:
Updated this logic to match the same logic found in `modelsToMap`.

The old logic caused a conflict between the `ServiceModel` found in `frontend/public/models/index.ts` vs the one found in `frontend/packages/knative-plugin/src/models.ts` and prevented the latter from being added to the `k8sModels` map.

Reply:
@christianvogt @spadgett this is the issue we are seeing in e2e tests. The `ServiceModel` in `frontend/packages/knative-plugin/src/models.ts` causes an issue going through the crd path (https://github.com/openshift/console/pull/2917/files#diff-5193194ffa917b7036f98babfc19dfa9R11)